### ISSUE TITLE
networkf netmail export

### ISSUE THREAD

**Issue by Wintermute740** [wwivbbs/wwiv#1129]
I am in three ftn-type networks, fsxnet, fidonet, and retronet (in that order
in init), and can't seem to get an email out anything other than the first.
For instance, I'm attempting to use AREAFIX on my upsteads. I address the
email properly for fsxnet, and networkf exports it fine and it goes out, and
all is good. I try to do the same with fidonet, addressing it in the same
manner, and I get the following error.

ERROR Unable to get address from to line: AREAFIX (1:340/0)

**Comment by wwiv** [comment]
I didn't realize a node number of 0 was valid FTN address, so we consider it an address that failed to parse.  Lemme look up the specs.

**Comment by wwiv** [comment]
Guess that's valid according to http://ftsc.org/docs/fts-0005.003
```
Field 2: Zone/Region/Net/Node number

    This field contains only numeric digits and is a number in the
    range of 0 to 32767. 
```

**Comment by Wintermute740** [comment]
I never realized, but, yes, my upstream node for retronet is also 0.

**Comment by wwiv** [comment]
Actually wwiv is wrong, we should consider this a valid node.

**Comment by wwiv** [comment]
Fixed in abbbf0e5657d502256e9a864ee9bb0b40882720c for wwiv 5.4
